src/wash/vector.hpp

```diff
@@ -119,4 +123,18 @@ namespace wash {
     };
 
     using SimulationVecT = Vec<double, DIM>;
+    
+    typedef Vec<double, 2> Vec2D;
+    typedef Vec<double, 3> Vec3D;
 }
+
+template <typename T, int dim>
+std::ostream& operator<<(std::ostream& s, const wash::Vec<T, dim>& vec) {
```

Comment:
Is there a reason why this is outside the `wash` namespace?

Reply:
This is just providing an override for the generic `<<` operator for the `wash::Vec` type. I don't think it makes sense for it to be namespaced